examples/05-operator-alt-image.yaml

```diff
@@ -16,9 +16,14 @@ spec:
       serviceAccountName: console-operator
       containers:
       - name: console-operator
-        # image: quay.io/benjaminapetersen/console-operator:v4.0.0.6
-        # image: quay.io/benjaminapetersen/console-operator-2:v4.1.0
-        image: quay.io/benjaminapetersen/console-operator-2:v4.1.2
+        # for development
```

Comment:
Was this intended to be included?  I believe the change to this file has already merged, may need a rebase.

Reply:
no, removed